### ISSUE TITLE
Option to disable show simplified view bar

### ISSUE THREAD

**Issue by oshanz** [brave/browser-android-tabs#2439]
## Description 

In android brave app "show simplified view" is so annoying and useless to me.
I'll attach screenshot how bad it is.

So I used to disable that from chrome flages. Using the setting reader mode.
```
chrome://flags/#enable-reader-mode
```
But I update to latest brave and now again I can see that bar popping.
And chrome 79 seems remove that option to disable reader mode.
Please give and option to disable show simplified view button.

![Screenshot_20200113-075505](https://user-images.githubusercontent.com/5364479/72233398-f80ecf80-35be-11ea-8ebc-2e6a698095b4.png)
![Screenshot_20200113-075440](https://user-images.githubusercontent.com/5364479/72233405-ff35dd80-35be-11ea-8b91-a91a48db5bd6.png)

## Brave version (brave://version info)

Brave | 1.6.2 chrome: 79.0.3945.93 (Official Build) (32-bit)
-- | --
OS | Android 7.1.1; SM-J510FN Build/NMF26X

**Comment by boggle247** [comment]
How do I easily copy my browser info here? I have the same issue as oshanzand would like to contribute but "brave://version info" just starts a web search when put in the address bar.

**Comment by boggle247** [comment]
how can we bring more attention to this issue? Do I make another duplicate issue? This makes the browser nearly unusable since I browse reddit from within brave and this pop up bar is ALWAYS there. I'd even be okay with a workaround

**Comment by oshanz** [comment]
@boggle247  one solution is, turn off automatic update and install brave version 1.6.1 from some Android APK provider. then you can toggle the `reader mode flag` as mention in the description.

this seems chromium developers decision that remove the flag from chrome core version 79 onwards.

for me worst thing is that bar is even to not working properly. as shown in the screenshots.

**Comment by boggle247** [comment]
Exactly. I tried to use it and it just breaks the page. I'd rather not stay on an unmaintained apk though. Perhaps I'll use one of the forks of brave or some other browser all together. It's super annoying.

**Comment by oshanz** [comment]
ok Its not fixed. I found this on some webpages again today.

* https://www.healthline.com/health-news/high-tempo-music-may-help-your-workout
* https://pradeeploganathan.com/architecture/event-driven-architecture/

**Comment by srirambv** [comment]
Whats the behaviour on Chrome? Could you try this on Brave-Beta and check if it reproduces on beta? You can download the Beta version via [Play Store](https://play.google.com/store/apps/details?id=com.brave.browser_beta&hl=en)

**Comment by boggle247** [comment]
This issue is fixed for me with the beta version

**Comment by srirambv** [comment]
Thanks for the confirmation @boggle247 . @oshanz mind giving Beta build a try and see if it is fixed for you as well. It looks like we dont need to do anything to fix this and since we're moving to new code base I'd like to close this issue as wontfix
